control/heading_hold/include/heading_hold/HHpid.h

```diff
@@ -0,0 +1,26 @@
+#ifndef HH_PID_H
+#define HH_PID_H
+
+#include <ros/ros.h>
```

Comment:
We should sort out the unnecessary duplicate PID classes.

Reply:
Agreed, though I think this can be added as an issue and solved later.